test/hemi.tokenlist.test.js

```diff
@@ -89,8 +89,13 @@ describe("List of tokens", function () {
           .replace(".e", "")
           .replaceAll(" ", "-")
           .toLowerCase();
-        assert.equal(logoURI, `${repoUrl}/master/src/logos/${filename}.svg`);
-        fs.accessSync(`src/logos/${filename}.svg`);
+
+        const filepath = logoURI.match(
+          // eslint-disable-next-line no-useless-escape
+          new RegExp(`^${repoUrl}/master/src/logos/${filename}\.(svg|png)$`),
```

Comment:
Try escaping the backslash 😅 

```suggestion
          new RegExp(`^${repoUrl}/master/src/logos/${filename}\\.(svg|png)$`),
```

Reply:
done